docs/source/jupyter/ontology-interface.ipynb

```diff
@@ -210,21 +579,32 @@
    "source": [
     "## Testing the type of the entities\n",
     "\n",
-    "In the OSP-core YAML ontology files three types of entities can be defined: classes, relationships and attributes. Relationships are subclasses of CUBA.RELATIONSHIP and attributes are subclasses of CUBA.ATTRIBUTE. There are different Python objects for the different entity types. You can use both to check which type of entity you are dealing with:"
+    "In the [YAML ontology installation files](../yaml.md), three types of entities can be defined: classes, relationships and attributes. OSP-core has its own vocabulary, the [CUBA namespace](../yaml.html#the-cuba-namespace), which describes, among other things, such entity types. Relationships are subclasses of `CUBA.RELATIONSHIP` and attributes are subclasses of `CUBA.ATTRIBUTE`. There are different Python objects for the different entity types. You can use both to check which type of entity you are dealing with:"
```

Comment:
As before: it seems the wrong link is given. `../owl.html` is what is meant perhaps (rather then `../yaml.md`)

Reply:
Here I am unsure as well. I think the problem is that it does not make sense to have two separate pages referring to the same concept, even if it is used for different purposes.

I will actually remove the link here.